qiita_pet/handlers/study_handlers/tests/test_base.py

```diff
@@ -59,5 +59,37 @@ def test_get(self):
         self.assertNotEqual(response.body, "")
 
 
+class TestStudyGetTags(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/study/get_tags/')
+        exp = ('{"status": "success", "message": "", "tags": '
+               '{"admin": [], "user": []}}')
+        self.assertEqual(response.code, 200)
+        self.assertEqual(response.body, exp)
+
+
+class TestStudyTags(OauthTestingBase):
+    def test_get(self):
+        response = self.get('/study/tags/1')
+        exp = ('{"status": "success", "message": "", "tags": []}')
+        self.assertEqual(response.code, 200)
+        self.assertEqual(response.body, exp)
+
+    def test_patch(self):
```

Comment:
I think this is logically distinct. The goal of a negative test here, and with `test_get`, is to verify the status codes and response messages are rational for malformed or inappropriate REST requests